.eslintignore

```diff
@@ -0,0 +1 @@
+Apps/
```

Comment:
I am OK with this as fixing eslint rules for this app could take a lot of work. 

To prevent a lot of future work, could you change this to only ignoring this FCM sample app? Keep APN app linted since it is currently? 

Also, could you create a new ticket to fix this in the future?

Reply:
Just to be clear, the FCM app is linted. But it uses the file and rules [generated by react native template](https://github.com/customerio/customerio-reactnative/pull/182/commits/5a5c39a62027b37e7ebda566c2d1285c686252ef#diff-e367ea0f93f62f918692b26ce1b1d69b79205bbd50a77adbb88e9504b5743d77) which probably causes the problem. I tried it again and [could see the following error](https://github.com/customerio/customerio-reactnative/actions/runs/5806288457/job/15738877632?pr=182)

```
ESLint couldn't find the config "@react-native" to extend from. Please check that the name of the config is correct.
```

I have updated `eslintignore` to [ignore FCM only](https://github.com/customerio/customerio-reactnative/pull/182/commits/a8ef672f948185ab2716148f90c7bfaa2e1e8c82) and removed `eslintIgnore` from `package.json`. Created [ticket](https://github.com/customerio/issues/issues/11030) for this which should fix this for whole project.